### ISSUE TITLE
QT 5.9 undefined reference to qt_version_tag@Qt_5.9

### ISSUE THREAD

**Issue by jaggzh** [cnr-isti-vclab/meshlab#453]
../distrib/libcommon.so: undefined reference to `qt_version_tag@Qt_5.9'
collect2: error: ld returned 1 exit status
Makefile:336: recipe for target '../distrib/meshlab' failed
make[1]: *** [../distrib/meshlab] Error 1

I get this with a Debian stable qmake:
QMake version 3.0   Using Qt version 5.7.1 in /usr/lib/x86_64-linux-gnu

I also get the same error if I use a python anaconda/miniconda version of qmake/libqt:
QMake version 3.1   Using Qt version 5.9.7 in /home/jaggz/miniconda3/lib

**Comment by jaggzh** [comment]
Okay, assuming my qt libs are actually proper versions, it seemed safe enough to bypass this version tagging test.  This isn't a good long-term solution, but if someone else is facing the problem and needs to just get around it, I just added " -DQT_NO_VERSION_TAGGING" to common/Makefile at the end of the DEFINES line.
This Makefile was apparently previously generated, possibly by one of the qmake or make commands I ran earlier when following the src/README.md.
In any case, after this it compiles and links and runs fine (distrib/meshlab runs without error).
